client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -164,11 +164,31 @@ class TransferDomainStep extends React.Component {
 		return domainProductPrice;
 	};
 
+	getProductPriceClass = () => {
+		const { cart, domainsWithPlansOnly, isSignupStep, selectedSite } = this.props;
+		const { searchQuery } = this.state;
+		const domainsWithPlansOnlyButNoPlan =
```

Comment:
What does this mean? That the domain is offered only if bundled with a plan?

Reply:
Yes, I modeled this after the `getProductPriceText()` function. If I'm reading it right, this returns true when a domain needs to be bundled with a plan but currently no plan is selected/active on the given site, or when we're in signup.

Reply:
👍